src/components/input/input.css

```diff
@@ -2,24 +2,56 @@
 @import "../../styles/spacing.css";
 @import "../../styles/typography.css";
 
+.container {
+  display: inline-block;
+  position: relative;
+}
+
+.label {
+  display: block;
+  font-family: var(--mavenlink-type-font-family);
```

Comment:
What happens if I'm a consumer and I want to use my own font stack while using MDS components? I think we have this same issue in Button. We're enforcing ourselves. I think we should have a discussion with Design and Product to ensure this is the approach we want to take. There's certainly an argument for both sides (forcing them to be more consistent, not forcing them to allow some flexibility and white labelling opportunities)

Reply:
__UPDATE: Nathan was able to confirm preference to go ahead and "force" use of Mavenlink font stack.__

So this is no longer an issue and we're good as we have here!

Reply:
Yup. It is also easy to use CSS to override our defaults.